File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/37_compito_python_20200124/Prenotazione.py
class Prenotazione:
    def __init__(self, codice_campo, ora_inizio):
        self._codice_campo = codice_campo
        self._ora_inizio = ora_inizio

    def __str__(self):
        return "(" + str(self._codice_campo) + ", " + str(self._ora_inizio) + ")"

File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Java/Exercises/37_compito_java_2020_01_24/Prenotazione.java
public class Prenotazione {
    protected int codiceCampo, oraInizio;

    public Prenotazione(int codiceCampo, int oraInizio) {
        this.codiceCampo = codiceCampo;
        this.oraInizio = oraInizio;
    }

    public String toString() {
        return "(" + codiceCampo + "," + oraInizio + ")";
    }
}